core/evancore.vue.libs/src/components/dapp-wrapper/dapp-wrapper.ts

```diff
@@ -495,7 +497,10 @@ export default class DAppWrapperComponent extends mixins(EvanComponent) {
         this.userInfo.mails = mails.slice(0, 5);
 
         // check the last read mail count against the current one, to check for new mails
-        const previousRead = parseInt(window.localStorage['evan-mail-read-count'] || 0, 10);
+        const previousRead = parseInt(
+          window.localStorage[`${this.getRuntime().activeIdentity}-mail-read-count`] || 0,
```

Comment:
Maybe we can refactor to the `localstorage.getItem()` syntax. There is no big difference, but most devs recommend to use the API: https://stackoverflow.com/a/24904863/1059828

Reply:
I forget it everytime, thanks for the suggestion, i changed all the localStorage accessors to the API.